src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -3,33 +3,150 @@
  * @author Box
  */
 import * as React from 'react';
-import flow from 'lodash/flow';
 import { FormattedMessage } from 'react-intl';
+import flow from 'lodash/flow';
+import getProp from 'lodash/get';
+import { MetadataEmptyState } from '@box/metadata-editor';
+import type { MessageDescriptor } from 'react-intl';
+import messages from '../common/messages';
+import { normalizeTemplates } from '../../features/metadata-instance-editor/metadataUtil';
+import { isUserCorrectableError } from '../../utils/error';
 import { withAPIContext } from '../common/api-context';
 import { withErrorBoundary } from '../common/error-boundary';
 import { withLogger } from '../common/logger';
-import { ORIGIN_METADATA_SIDEBAR_REDESIGN } from '../../constants';
-import { EVENT_JS_READY } from '../common/logger/constants';
 import { mark } from '../../utils/performance';
-import messages from '../common/messages';
+import { EVENT_JS_READY } from '../common/logger/constants';
+import API from '../../api';
+import {
+    FIELD_IS_EXTERNALLY_OWNED,
+    FIELD_PERMISSIONS,
+    FIELD_PERMISSIONS_CAN_UPLOAD,
+    IS_ERROR_DISPLAYED,
+    ORIGIN_METADATA_SIDEBAR_REDESIGN,
+} from '../../constants';
+import type { ElementsXhrError, ErrorContextProps } from '../../common/types/api';
+import type { BoxItem } from '../../common/types/core';
+import type { MetadataEditor, MetadataTemplate } from '../../common/types/metadata';
+import type { WithLoggerProps } from '../../common/types/logging';
 
 import './MetadataSidebarRedesign.scss';
 
+interface ExternalProps {
+    isBoxAiSuggestionsFeatureEnabled: boolean;
```

Comment:
```suggestion
    isBoxAiSuggestionsEnabled: boolean;
```

I dont see a reason to put the word "Feature" in the prop name, seems descriptive enough without the word "Feature" in it

Reply:
changed that